### PR TITLE
Django 1.8 support

### PR DESCRIPTION
**Pull request by kaedroho**
This implements Django 1.8 support

### CODE REVIEW

File: modelcluster/models.py [hunk 1]
```diff
@@ -54,31 +70,33 @@ def model_from_serializable_data(model, data, check_fks=True, strict_fks=False):
     kwargs = {pk_field.attname: data['pk']}
     for field_name, field_value in data.items():
         try:
-            field = model._meta.get_field(field_name)
+            field = model._meta.get_field(field_name, many_to_many=False)
```
Comment (kaedroho) [inline modelcluster/models.py]: Would really like to get this working without that ``many_to_many`` flag.

**Comment by kaedroho** [comment]
Deprecation warnings fixed, tox setup added, and a few other tweaks. Ready for merge!

**Comment by gasman** [comment]
Did anyone get a chance to review this? I didn't...

**Comment by kaedroho** [comment]
Nope, didn't think you wanted to. Feel free to review now though

**Pull request merged**